### PR TITLE
Add 4.5.0 compatible AppSupport instructions

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: Develop/Platform/Testing_Advice/README.md [hunk 1]
```diff
@@ -60,7 +60,10 @@ More information about logs below:
 
 ## Android apps
 * `/system/bin/logcat`
-* `devel-su lxc-attach -n aliendalvik /system/bin/logcat`
+* On Sailfish OS 4.4.0:
+    * `devel-su lxc-attach -n aliendalvik /system/bin/logcat`
+* On Sailfish OS 4.5.0:
+    * `devel-su lxc-attach -n aliendalvik --lxcpath=/tmp/appsupport /system/bin/logcat`
```
Comment (mkosola) [inline Develop/Platform/Testing_Advice/README.md:66]: this should be "devel-su appsupport-attach /system/bin/logcat"

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh [hunk 2]
```diff
@@ -112,7 +112,7 @@ else
     # The next block covers Xperia 10, 10 II, 10 III, and XA2
     if [ "${MODELSTR:14:10}" = "Xperia 10 " ] || [ "${MODELSTR:14:10}" = "Xperia XA2" ]; then 
         # Longer command if release is 4.5. or later
-        LOGCATCMD="lxc-attach -q -n aliendalvik --lxcpath=/tmp/aliendalvik -- /system/bin/logcat -d"
+        LOGCATCMD="lxc-attach -q -n aliendalvik --lxcpath=/tmp/appsupport -- /system/bin/logcat -d"
```
Comment (mkosola) [inline Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:115]: Please use "appsupport-attach /system/bin/logcat -d"

**Comment by vigejolla** [comment]
There are some other places which had updated instructions already before, I'll need to change them as well to use appsupport-attach

**Comment by vigejolla** [comment]
Fixed all lxc-attach invocations to use appsupport-attach instead

**Review by mkosola — APPROVED** [review]

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh [hunk 3]
```diff
@@ -112,7 +112,7 @@ else
     # The next block covers Xperia 10, 10 II, 10 III, and XA2
     if [ "${MODELSTR:14:10}" = "Xperia 10 " ] || [ "${MODELSTR:14:10}" = "Xperia XA2" ]; then 
         # Longer command if release is 4.5. or later
-        LOGCATCMD="lxc-attach -q -n aliendalvik --lxcpath=/tmp/aliendalvik -- /system/bin/logcat -d"
+        LOGCATCMD="appsupport-attach /system/bin/logcat -d"
```
Comment (jovirkku) [inline Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:115]: This command does its job but lxc-attach prints some warnings:

```
9. Android logs
- logcat log
- status of Android App Support
lxc-attach: aliendalvik: terminal.c: lxc_terminal_create_native: 927 Not a directory - Failed to open terminal multiplexer device
lxc-attach: aliendalvik: af_unix.c: lxc_abstract_unix_recv_fds_iov: 318 Invalid argument - Received no file descriptors
lxc-attach: aliendalvik: commands.c: lxc_cmd_rsp_recv_fds: 129 Failed to receive file descriptors
lxc-attach: aliendalvik: af_unix.c: lxc_abstract_unix_recv_fds_iov: 318 Invalid argument - Received no file descriptors
lxc-attach: aliendalvik: commands.c: lxc_cmd_rsp_recv_fds: 129 Failed to receive file descriptors

Done. All logs collected.
```
Would there be any way to silence them?

Comment (mkosola) [reply Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:115]: If we silence those warnings then we don't see real errors...

**Pull request merged**